src/physics/material/detail/Utils.cc

```diff
@@ -50,19 +98,12 @@ real_type calc_mass_rad_coeff(const ElementDef& el)
             lrad_prime = std::log(1194.0 * std::pow(z_real, -2.0 / 3));
     }
 
-    // Eq 33.27 for calculating f(Z), the coulomb correction factor
-    real_type alphazsq = alpha_fine_structure * std::min(el.atomic_number, 92);
-    alphazsq *= alphazsq;
-    const real_type fz = alphazsq
-                         * (1 / (1 + alphazsq) + 0.20206 - 0.0369 * alphazsq
-                            + 0.0083 * alphazsq * alphazsq
-                            - 0.002 * alphazsq * alphazsq * alphazsq);
-
     // Eq 33.26
```

Comment:
```suggestion
    // Eq 34.25
```

Reply:
Point taken... after this can we please never update the RPP reference again? 😂